server/ServerFSM.cpp

```diff
@@ -748,6 +748,14 @@ void MPLobby::ValidateClientLimits() {
             ai_count++;
     }
 
+    const int human_connected_count = human_count;
+
+    // for load game consider as human all non-AI empires
+    // because human player could connect later in game
+    if (!m_lobby_data->m_new_game) {
+        human_count = m_lobby_data->m_save_game_empire_data.size() - ai_count;
```

Comment:
does this matter? if there are fewer players than empires, won't it already prevent the game from starting?